### ISSUE TITLE
Horizontal form control-label's padding-top is off on xs-sized screens

### ISSUE THREAD

**Issue by frehner** [twbs/bootstrap#18727]
Searched for duplicates, didn't find any. Sorry if this has been mentioned before.

1a5fb4e appears to break the vertical centering of a horizontal form when using col-xs and viewing on xs-sized screens.  Notice in forms.less that .control-label's padding-top is only applied on >=sm sized screens.  

Example: https://jsfiddle.net/kjng4fmp/ and resize down to a xs-sized screen to see the issue of the label text on the left not being vertically aligned with the input text on the right.

Was this intentional?  If so, can you put something in the documentation that horizontal forms no longer work when using col-xs?  If not, then can we see about fixing this?

**Comment by twbs-lmvtfy** [comment]
Hi @frehner!

You appear to have posted a live example (http://s.bootply.com/render/iZdLOlK5Zf), which is always a good first step. However, according to [Bootlint](https://github.com/twbs/bootlint), **your example has some Bootstrap usage errors**, which might potentially be causing your issue:
* line 10, column 9: [W013](https://github.com/twbs/bootlint/wiki/W013): Bootstrap version might be outdated. Latest version is at least 3.3.6 ; saw what appears to be usage of Bootstrap 3.3.5
* line 102, column 9: [W013](https://github.com/twbs/bootlint/wiki/W013): Bootstrap version might be outdated. Latest version is at least 3.3.6 ; saw what appears to be usage of Bootstrap 3.3.5

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by frehner** [comment]
Thanks for the heads up, bot. Sadly bootply doesn't appear to have 3.3.6 on their site yet. However, that shouldn't matter since this was broken in the 3.2.0 release.

**Comment by frehner** [comment]
Though to make you happy, I made a jsfiddle with 3.3.6 https://jsfiddle.net/kjng4fmp/

**Comment by mdo** [comment]
[**Bootstrap 3 is no longer being officially developed or supported.**](https://github.com/twbs/bootstrap/issues/20631)

All work has moved onto our next major release, v4. As such, this issue or pull request is being closed as a "won't fix." For additional help and support, we recommend [utilizing our community resources](https://github.com/twbs/bootstrap/blob/master/README.md#community). Thanks for your understanding, and see you on the other side of v4!

<3,
@mdo and team
